script/Ciel/Day 8/08_01_CIEL08_4.txt

```diff
@@ -236,7 +236,7 @@ As I sit here on the bench, she's prancing around in circles in front of me.
 [sha:a77068eecb3b41b660f6c2c6540efac583ba12d6]{
 -- Page 351, Offset 19132.
 -- ……なんだろう、アレ。
-What's she doing...?
+...What's she doing?
```

Comment:
undo, we're generally trying to get rid of these unless they make a lot of sense